doc/classes/Camera2D.xml

```diff
@@ -364,6 +375,8 @@
 		<member name="smoothing_speed" type="float" setter="set_follow_smoothing" getter="get_follow_smoothing">
 		</member>
 		<member name="zoom" type="Vector2" setter="set_zoom" getter="get_zoom">
+			The zoom value of the camera relative to the viewport. Use [code]Vector2(0.5,0.5)[/code] to make the camera zoom in 2x the scene or [code]Vector2(2,2)[/code] for a 2x zoom out.
```

Comment:
First sentence: "The camera's zoom relative to the viewport."

Examples need to be rewritten. I suggest something like:

    Values larger than [code]Vector2(1, 1)[/code] zoom out and smaller values zoom in. For example, use [code]Vector2(0.5, 0.5)[/code] for a 2x zoom in.